Misc/NEWS.d/next/Core and Builtins/2023-08-22-09-57-59.gh-issue-106016.lpbUq8.rst

```diff
@@ -0,0 +1,2 @@
+Add support for module ``__setattr__`` and ``__delattr__``. Patch by Sergey
+B Kirpichev.
```

Comment:
It's worth to document these new features at: https://docs.python.org/dev/whatsnew/3.13.html#other-language-changes